subprojects/docs/src/docs/userguide/authoring-builds/organizing_gradle_projects.adoc

```diff
@@ -95,8 +95,12 @@ Gradle tries to locate a `settings.gradle` (Groovy DSL) or a `settings.gradle.kt
 For that purpose, the runtime walks the hierarchy of the directory tree up to the root directory.
 The algorithm stops searching as soon as it finds the settings file.
 
+A multi-project build must have a `settings.gradle` file in the root project of the multi-project hierarchy.
+It is required because the settings file defines which projects are taking part in a <<multi_project_builds.adoc#multi_project_builds,multi-project build>>.
+For a single-project build, the settings file is optional.
```

Comment:
IIRC for a single-project build, the settings file is mandatory too.

Reply:
Who can we ask to validate the truthiness of that statement? If so, I can eliminate a lot of verbosity and just replace it with "the settings file is mandatory".

Or maybe it's better to simplify the docs and say that regardless, since omitting a settings file supposedly has performance implications.

Reply:
My bad. We intended to make it mandatory at some point but it currently is not.
I think we should recommend to always have one though.

Reply:
Two sentences after this: 
> Always add a `settings.gradle` to the root directory of your build to avoid the initial performance impact.

I'll move the recommendation up on the page a bit, I think you're right that we shouldn't bury the lede.